src/app/routes/ewt-patron/withdraw/withdraw.component.html

```diff
@@ -1,18 +1,35 @@
-<ng-container *ngIf="(isWithdrawingDelayFinished$ | async) else withdrawTemplate">
-  <mat-progress-bar
-      color="primary"
-      mode="indeterminate">
-  </mat-progress-bar>
-  <p class="text-center mt-2">Preparing your withdrawal…</p>
-</ng-container>
-<ng-template #withdrawTemplate>
-  <div class="confirm-wrapper">
-    <p class="text-center">You are going to withdraw your tokens. <br>Please confirm.</p>
-    <app-divider></app-divider>
-    <div class="d-flex justify-content-center mt-3">
-      <button mat-raised-button class="stake-btn text-color text-uppercase" (click)="withdraw()">Confirm</button>
-    </div>
+<div class="d-flex justify-content-center align-items-center flex-column stake-success-container">
+  <h3 class="mt-3 mb-4">Withdraw?</h3>
+  <p class="text-center mt-1">
+    EWT will be transfered to your account balance.
+    You will be charged with gass fee.
+  </p>
+</div>
+
+<mat-form-field class="staking-textfield" appearance="outline">
```

Comment:
Usually it is a good approach to keep all form fields inside a form, even if it's only one.